CXX_for_OpenCL.txt

```diff
@@ -0,0 +1,46 @@
+// Copyright 2019 The Khronos Group. This work is licensed under a
+// Creative Commons Attribution 4.0 International License; see
+// http://creativecommons.org/licenses/by/4.0/
+
+= The C++ for OpenCL Programming Language Documentation
+:R: pass:q,r[^(R)^]
+Khronos{R} OpenCL Working Group
+:data-uri:
+:icons: font
+:toc2:
+:toclevels: 2
+:max-width: 100
+:numbered:
+:imagewidth: 800
+:fullimagewidth: width="800"
+:source-highlighter: coderay
+:title-logo-image: image:images/OpenCL.png[Logo,200,200]
+
+:blank: pass:[ +]
+:cpp: pass:[C++]
+:cpp14: pass:[C++17]
+
+// Various special / math symbols. This is easier to edit with than Unicode.
+include::config/attribs.txt[]
+
+<<<<
+
+include::copyrights-ccby.txt[]
+
+// type of the source code in the document
+:language: {basebackend@docbook:c++:cpp}
+
+include::cxx4opencl/intro.txt[]
+
+<<<
```

Comment:
Minor comment: These are explicit page breaks that get inserted into the PDF, and I think they're causing the paging to be a little weird.  I'd try removing them and see if the paging works better.  I think you'll still want the one before the "Acknowledgements", but you might be able to remove the others.